packages/batcher/batcher-transaction-poster/src/avail.ts

```diff
@@ -0,0 +1,35 @@
+import type { Pool } from 'pg';
+import { base64Encode } from '@polkadot/util-crypto';
+import BatchedTransactionPosterBase from './transactionPoster.js';
+
+class AvailBatchedTransactionPoster extends BatchedTransactionPosterBase {
```

Comment:
we will need to add information about how to use the Avail batcher to the Paima docs